platform/linux-generic/pktio/dpdk.c

```diff
@@ -1505,20 +1514,17 @@ static int dpdk_pktio_term(void)
 }
 
 static void prepare_rss_conf(pktio_entry_t *pktio_entry,
-			     const odp_pktin_queue_param_t *p)
+			     const odp_pktin_queue_param_t *p, struct rte_eth_dev_info *dev_info)
```

Comment:
On second thought the parameters could be e.g. `uint64_t rss_offloads` as the whole struct is not  necessary.